server/src/modules/clubs/mappers/club-map.ts

```diff
@@ -1,21 +1,58 @@
 import { UniqueEntityID } from '../../../shared/domain/unique-entity-id'
 import { ClubEntity } from '../../../shared/infra/db/entities/club.entity'
-import { Club } from '../domain/entities/club'
-import { ClubDTO } from './club-dto'
+import { EventEntity } from '../../../shared/infra/db/entities/event.entity'
+import { BasicEvent, Club } from '../domain/entities/club'
+import { ClubDTO, BasicEventDTO } from './club-dto'
+
+let basicEventMap = async (events: Array<EventEntity>) => {
```

Comment:
Can this be const?

```suggestion
const basicEventMap = async (events: Array<EventEntity>) => {
```

Also I'm guessing we don't want this method to be in a fully-fledged Map class because `BasicEvent` isn't technically a domain object?

Reply:
I could make a separate file for it, but this Map class is the only thing that uses it, so I thought might as well just have it at the top. Made const.